static/app/views/replays/detail/breadcrumbs/breadcrumbItem.tsx

```diff
@@ -44,6 +46,20 @@ function BreadcrumbItem({
     (e: React.MouseEvent<HTMLElement>) => onClick?.(crumb, e),
     [onClick, crumb]
   );
+  const handleIconClick = useCallback(() => {
+    switch (crumb.type) {
+      case 'navigation':
+      case 'debug':
+        setActiveTab('network');
+        break;
+      case 'ui':
+        setActiveTab('dom');
+        break;
+      default:
+        setActiveTab('console');
+        break;
+    }
```

Comment:
is there an error type too we could link?

Reply:
Unfortunately errors aren't coming through on new replays but yeah should it link to console or issues ?

Reply:
errors should link to Issues i think.

and console still makes a great default. 

cc @jas-kas this is part of that table we were talking about yesterday. what's what and what goes where.